src/PrestaShopBundle/Controller/Admin/Sell/Address/AddressController.php

```diff
@@ -401,6 +404,90 @@ public function editAction(int $addressId, Request $request): Response
         ]);
     }
 
+    /**
+     * Handles edit form rendering and submission for order address
+     *
+     * @AdminSecurity(
+     *     "is_granted('update', request.get('_legacy_controller'))",
+     *     redirectRoute="admin_orders_index"
+     * )
+     *
+     * @param int $orderId
+     * @param string $addressType
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function editOrderAction(int $orderId, string $addressType, Request $request): Response
+    {
+        try {
+            $order = new Order($orderId);
+            $addressId = null;
+            switch ($addressType) {
+                case 'delivery':
+                    $addressType = OrderAddressType::DELIVERY_ADDRESS_TYPE;
+                    $addressId = $order->id_address_delivery;
+                    break;
+                case 'invoice':
+                    $addressType = OrderAddressType::INVOICE_ADDRESS_TYPE;
+                    $addressId = $order->id_address_invoice;
+                    break;
+            }
+
+            /** @var EditableCustomerAddress $editableAddress */
+            $editableAddress = $this->getQueryBus()->handle(new GetCustomerAddressForEditing((int) $addressId));
+
+            $addressFormBuilder = $this->get(
+                'prestashop.core.form.identifiable_object.builder.address_form_builder'
+            );
+            // Special order handler
+            $addressFormHandler = $this->get(
+                'prestashop.core.form.identifiable_object.handler.order_address_form_handler'
+            );
+
+            // Address type required for EditOrderAddressCommand
+            $formData = [
+                'address_type' => $addressType,
+            ];
+            // Country needs to be preset before building form type because it is used to build state field choices
+            if ($request->request->has('customer_address') && isset($request->request->get('customer_address')['id_country'])) {
+                $formCountryId = (int) $request->request->get('customer_address')['id_country'];
+                $formData['id_country'] = $formCountryId;
+            }
+
+            // Addres form is built based on address id to fill the data related to this address
+            $addressForm = $addressFormBuilder->getFormFor($addressId, $formData);
+            $addressForm->handleRequest($request);
+
+            // Form is handled based on Order ID because that's the order that needs update
+            $result = $addressFormHandler->handleFor($orderId, $addressForm);
+
+            if ($result->isSubmitted() && $result->isValid()) {
+                $this->addFlash('success', $this->trans('Successful update.', 'Admin.Notifications.Success'));
```

Comment:
Done

Reply:
I only changed them in this controller, but there are many other use of this wrong wording in other controllers I don't want to make all the updates in this PR because it's not related to the issue, but should we change all the present wordings in another PR to be consistent everywhere?

Reply:
Absolutely! I think a part of it has already been done in PR #17819, can you check and let me know if all controllers are changed? If not, I'll create an issue to have it all changed. :-)